sycl/include/CL/sycl/detail/image_accessor_util.hpp

```diff
@@ -99,21 +100,22 @@ getImageOffset(const vec<T, 4> &Coords, const id<3> ImgPitch,
 
 // Process cl_float4 Coordinates and return the appropriate Pixel Coordinates to
 // read from based on Addressing Mode for Nearest filter mode.
-cl_int4 getPixelCoordNearestFiltMode(cl_float4, const addressing_mode,
-                                     const range<3>);
+SYCL_API cl_int4 getPixelCoordNearestFiltMode(cl_float4, const addressing_mode,
```

Comment:
Off-topic. Probably we could move modified functions in this file to source dir.